app/src/sharedTest/java/org/oppia/app/faq/FAQSingleActivityTest.kt

```diff
@@ -95,6 +122,24 @@ class FAQSingleActivityTest {
     }
   }
 
+  @Test
+  fun openFAQSingleActivity_checkAnswer_isCorrectlyParsed() {
+    val answerTextView = activityTestRule.activity.findViewById(
+      R.id.faq_answer_text_view
+    ) as TextView
+    val htmlParser = htmlParserFactory.create(
+      resourceBucketName,
+      /* entityType= */ "",
```

Comment:
should we add some comments here why we are sending empty strings for suture reference?

Reply:
Done.